messages/config-serialize-function.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+module.exports = function({ key, objectKey }) {
+
+    // special case for parsers
+    const isParser = objectKey === "parser" && key === "parse";
```

Comment:
```suggestion
    const isParser = objectKey === "parser" && (key === "parse" || key === "parseForESLint");
```

Custom parsers more often provide `parseForESLint()` instead of `parse()`.